backend/app/helpers/spree/admin/base_helper.rb

```diff
@@ -218,6 +218,16 @@ def admin_logout_link
         end
       end
 
+      def stores_formatted_for_select
```

Comment:
@MatthewKennedy let's add this to `Spree::Admin::StoresHelper` - there will be more stores-related methods